1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,39 @@
-function setAlarm() {}
+function displayRemainingTime(totalSeconds) {
+  // 1. we get the html element for the time remaining part
+  // 2. we want to figure out what this new string would be eg calucuate if it is 100
+  // 3. use the value in step 2 and put it in the html element in step 1
+  
+    let timeRemainingString = document.getElementById("timeRemaining");
+    let minutes = Math.floor(totalSeconds / 60); // "Time Remaining: 00:00";
+    let seconds = totalSeconds % 60;
+
+    let formattedMinutes = ("0" + minutes).slice(-2);
+    let formattedSeconds = ("0" + seconds).slice(-2);
+
+    timeRemainingString.textContent =
+      "Time Remaining: " + formattedMinutes + ":" + formattedSeconds;
+} 
+
+function setAlarm() {
+  let inputElement = document.getElementById("alarmSet");
```

Comment:
it would be preferrable to use `const` for variables that do not change. This is especially important in cases of html elements, because we would rarely want that variable to be re-assigned a different value.

In larger code bases, it's important to follow const/let rules correctly.

Reply:
Thank you for your valuable advice and your kind and supportive words in your reviews! I have corrected the variables.